Jenkinsfile

```diff
@@ -324,6 +341,47 @@ pipeline {
           // FIXME can't be defined globally due to a bug in Jenkins:
           // https://issues.jenkins-ci.org/browse/JENKINS-49826
           matrixBuilder = { def matrix, int level ->
+            variations.failFast = true
+            variations["slurm"] = {
+              try {
+                node("KNL_PRIO") {
+                  parallel "allocation": {
+                    try {
+                      timeout(time: 2, unit: 'HOURS') {
+                        // Allocate a new job
+                        sh """
+                          salloc --job-name=mardyn-test --nodes=1-2 \
+                            --nodelist=mpp3r03c05s01,mpp3r03c05s02\
+                            --tasks-per-node=3 --time=02:00:00 --begin=now+150\
+                            sleep 7200 || echo 0
+                            sleep 7200
+                        """
+                      }
+                    }
+                    catch (err) {
+                      println err
+                    }
+                  }, "slurmcontrol": {
+                    sleep 150
+                    // Store jobid
+                    knl_jobid = sh(
+                      returnStdout: true,
+                      script: 'squeue -O jobid | sed -n 2p'
+                    ).replace("\n", "")
+                    println "Scheduled job " + knl_jobid
+                    while (results.count { key, value -> key.contains("KNL") } < variations.count { key, value -> key.contains("KNL") }) {
+                      sleep 120
+                    }
+                    sh "scancel $knl_jobid -f --user=ga38cor3"
+                    error "all finished"
```

Comment:
should probably not be an error

Reply:
This is a hack. I'm using the error to kill the running slurm step via failfast. I'll add comments to explain what's going on.